### PR TITLE
[REF] [7.0] pylint;

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

File: tree_view_record_id/orm.py [hunk 1]
```diff
@@ -12,12 +12,9 @@
...
 
-""" procedural code is executed even if the module is not installed
-
-    TRY TO SWITCH to _register_hook() in the model to avoid
-    execution when not installed in V 8.0 version
-
-"""
+# procedural code is executed even if the module is not installed
+#    TRY TO SWITCH to _register_hook() in the model to avoid
+#    execution when not installed in V 8.0 version
```
Comment (guewen) [inline tree_view_record_id/orm.py]: It is valid to have a docstring in a python module, but it must be before python code, so before the imports.

File: fetchmail_attach_from_folder/match_algorithm/email_domain.py [hunk 2]
```diff
@@ -20,10 +20,10 @@
 #
 ##############################################################################
 
-from email_exact import email_exact
+from . import email_exact
 
 
-class email_domain(email_exact):
+class email_domain(email_exact.email_exact):
```
Comment (guewen) [inline fetchmail_attach_from_folder/match_algorithm/email_domain.py]: Why not?

```
from .email_exact import email_exact
```

Anyway both work, you don't need to change.

**Comment by tafaRU** [comment]
@legalsylvain I think you should use the raw notation as suggested that is r prefix.

**Comment by legalsylvain** [comment]
Can you propose me the exact code, I don't know regex very well.
Thanks.

**Comment by tafaRU** [comment]
@legalsylvain,  raw string notation (r"text") keeps regular expressions sane....
In your case:

```
attribute_match = re.search(
              r'([a-zA-Z_]+)=\%s', conf['ldap_filter'])
```

File: tree_view_record_id/orm.py (hunk 1 above)
Comment (legalsylvain) [inline tree_view_record_id/orm.py]: Done.

File: fetchmail_attach_from_folder/match_algorithm/email_domain.py (hunk 2 above)
Comment (legalsylvain) [inline fetchmail_attach_from_folder/match_algorithm/email_domain.py]: Thanks, I didn't know.

**Comment by legalsylvain** [comment]
@tafaRU: Thanks.
Done.

**Comment by guewen** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by tafaRU** [comment]
Once it's green, :+1:

**Pull request merged**